Makefile

```diff
@@ -195,10 +195,16 @@ test:
 	INVENTORY=$(shell $(call get_service,assisted-service) | sed 's/http:\/\///g') \
 		DB_HOST=$(shell $(call get_service,postgres) | sed 's/http:\/\///g' | cut -d ":" -f 1) \
 		DB_PORT=$(shell $(call get_service,postgres) | sed 's/http:\/\///g' | cut -d ":" -f 2) \
+		OCM_HOST=$(shell $(call get_service,wiremock) | sed 's/http:\/\///g') \
 		TEST_TOKEN="$(shell cat $(BUILD_FOLDER)/auth-tokenString)" \
+		TEST_TOKEN_ADMIN="$(shell cat $(BUILD_FOLDER)/auth-tokenAdminString)" \
+		TEST_TOKEN_UNALLOWED="$(shell cat $(BUILD_FOLDER)/auth-tokenUnallowedString)" \
 		ENABLE_AUTH="true" \
 		go test -v ./subsystem/... -count=1 -ginkgo.focus=${FOCUS} -ginkgo.v -timeout 30m
 
+deploy-wiremock: deploy-namespace
+	python3 ./tools/deploy_wiremock.py --target $(TARGET)
```

Comment:
namespace probably should be a parameter.

Reply:
+1